compiler/src/test/kotlin/io/spine/protodata/AstTest.kt

```diff
@@ -105,10 +105,29 @@ class `AST extensions should` {
             assertThat(className)
                 .isEqualTo("${TestEnv.packageName}.${TestEnv.outerClassName}$${simpleName}")
         }
+
+        @Test
+        fun enum() {
+            val simpleName = "ExplosiveType"
+            val type = EnumType
+                .newBuilder()
```

Comment:
We can already have `.newBuilder()` on the same line with type. Latest IDEA versions have a formatting setting that covers that, and this setting is turned on in `config`.

Reply:
Chain call alignment does not work for Kotlin yet. We'd have something like
```kotlin
val type = EnumType.newBuilder()
    .setField(..)
    .setOther(..)
```
instead of
```kotlin
val type = EnumType.newBuilder()
                   .setField(..)
                   .setOther(..)
```

I suggest wrapping first method calls until Kotlin code style config catches up with the Java one. If you'd like, let's discuss in person tomorrow.